### DISCUSSION TITLE
adding "WebChatGPT" to the program

### DISCUSSION THREAD

**Discussion by amrpyt** [lencx/ChatGPT#261] [Ideas]
**"WebChatGPT"🔗(https://github.com/qunash/chatgpt-advanced)**

Dear Developers,

I would like to suggest that you consider adding this browser extension "WebChatGPT" to your current suite of tools. I would love to hear your thoughts on this idea and see how it could be integrated into your development workflow.

Thanks for your consideration.

**Comment by ThereThatGuy** [comment]
This is an excellent addon that makes the model much more robust IMO. I'd also love this feature.
